addons/new-osio-addon/src/test/java/io/fabric8/launcher/osio/steps/OpenShiftStepsTest.java

```diff
@@ -28,13 +32,15 @@
 import org.junit.contrib.java.lang.system.ProvideSystemProperty;
 import org.junit.rules.RuleChain;
 
-import static io.fabric8.launcher.osio.HoverflyRuleConfigurer.createHoverflyProxy;
-
 public class OpenShiftStepsTest {
 
     @ClassRule
     public static RuleChain ruleChain = RuleChain
-            .outerRule(new ProvideSystemProperty("https.proxyHost", "127.0.0.1")
+            // Set real environment variables when recording.
+            // When recording is done, adapt and test that the Hoverfly descriptors are working in simulation mode with the simulation environment.
```

Comment:
I'm going to approve although these comments are a bit unclear to me.

Reply:
You shouldn't have approved if they are unclear. lol :)
It means after recording with real credentials (environment) on a real external service, you should test that it work in simulation mode with mock credentials.

If you have a proposition to make them clearer I would gladly create a new PR because they are at 4 different places.

Reply:
You know you can push more commits to this PR right? :)

Reply:
I thought it was already merged, but it was the other PR --'

Reply:
@tako it is better?
```Java
// After recording on a real environment against a real service,
// You should adapt the Hoverfly descriptors (.json) to make them work in simulation mode with the mock environment.
```